packages/core/src/VRange.ts

```diff
@@ -240,7 +240,7 @@ export class VRange {
         } else if (position === RelativePosition.AFTER) {
             reference = reference.lastLeaf();
         }
-        if (reference.is(ContainerNode) && !reference.hasChildren()) {
+        if (reference instanceof ContainerNode && !reference.hasChildren()) {
```

Comment:
Du coup faudrait retirer la fonction `is` de `AbstractNode` si on décide qu'on ne veut plus s'en servir.